fuels-signers/src/wallet/mod.rs

```diff
@@ -1,15 +1,17 @@
+use crate::provider::{Provider, ProviderError};
```

Comment:
`src/wallet/mod.rs` is the outdated Rust 2015 style. We should just have a `src/wallet.rs` file instead.